doc/package-notes.xml

```diff
@@ -814,4 +814,46 @@ citrix_receiver.override {
    </para>
   </section>
  </section>
+ <section xml:id="sec-ibus-typing-booster">
+  <title>ibus-engines.typing-booster</title>
+
+  <para>This package is an ibus-based completion method to speedup typing.</para>
+
+  <section xml:id="sec-ibus-typing-booster-activate">
+   <title>Activate engine</title>
+
+   <para>
+    IBus needs to be configured accordingly to activate <literal>typing-booster</literal>. The configuration
```

Comment:
I would add a comma after “IBus needs to be configured accordingly” clause but we should probably check with a native speaker.

Reply:
I'm not 100% sure here, but I thought that infinitive sentences at the end don't necessarily need the comma.